### ISSUE TITLE
source_id registration of INM-CM5-0

### ISSUE THREAD

**Issue by volodin1** [WCRP-CMIP/CMIP6_CVs#358]
model description is complete

**Comment by volodin1** [comment]
label = INMCM50
label_extended = INMCM50
source_id = INMCM50
institution_id = INM
release_year = 2016
activity_participation = [CMIP]

aerosol:
description = INMAER1
nominal_resolution = 150km
atmos:
description = INMAM50 (2x1.5, 180x120 longitude/latitude; 73 levels; top level sigma=0.0002)
nominal_resolution = 150 km
atmosChem:
description = none
nominal_resolution = none
land:
description = INMLND1
nominal_resolution = 150 km    
landIce:
description = none
nominal_resolution = none 
ocean:
description = INMOM5 (North Pole shifted to 60N, 90E. 0.5x0.25; 720 x 720 longitude/latitude; 40 levels; vertical sigma coordinate)
nominal_resolution = 40 km
ocnBgchem:
description = none
nominal_resolution = none
seaIce:
description = INMICE1
nominal_resolution = 40km

**Comment by durack1** [comment]
@volodin1 our currently registered model names are:
```
[u'ACCESS-1-0',
 u'AWI-CM-1-0-HR',
 u'AWI-CM-1-0-LR',
 u'AWI-CM-1-0-MR',
 u'BESM-2-7',
 u'BNU-ESM-1-1',
 u'CAMS_CSM1-0',
 u'CIESM',
 u'CMCC-CM2-HR4',
 u'CMCC-CM2-HR5',
 u'CMCC-CM2-SR5',
 u'CMCC-CM2-VHR4',
 u'CMCC-ESM2-HR5',
 u'CMCC-ESM2-SR5',
 u'CNRM-CM6-1',
 u'CNRM-CM6-1-HR',
 u'CNRM-ESM2-1',
 u'CNRM-ESM2-1-HR',
 u'CanESM5',
 u'EC-Earth3',
 u'EC-Earth3-AerChem',
 u'EC-Earth3-CC',
 u'EC-Earth3-GrIS',
 u'EC-Earth3-HR',
 u'EC-Earth3-LR',
 u'EC-Earth3-Veg',
 u'EC-Earth3-Veg-LR',
 u'EMAC-2-53-AerChem',
 u'EMAC-2-53-Vol',
 u'GFDL-AM4',
 u'GFDL-CM4',
 u'GFDL-ESM4',
 u'GISS-E2-1G',
 u'GISS-E2-1H',
 u'HadGEM3-GC31-HH',
 u'HadGEM3-GC31-HM',
 u'HadGEM3-GC31-LL',
 u'HadGEM3-GC31-MM',
 u'IITM-ESM',
 u'MIROC-ES2H',
 u'MIROC-ES2L',
 u'MIROC6',
 u'MPIESM-1-2-HR',
 u'MPIESM-1-2-LR',
 u'MPIESM-2-LR',
 u'MRI-ESM2-0',
 u'NICAM16-7S',
 u'NICAM16-8S',
 u'NICAM16-9D-L78',
 u'NICAM16-9S',
 u'NorESM2-HH',
 u'NorESM2-LM',
 u'NorESM2-LME',
 u'NorESM2-LMEC',
 u'NorESM2-MH',
 u'NorESM2-MM',
 u'PCMDI-test-1-0',
 u'UKESM1-0-LL',
 u'UKESM1-0-MMh',
 u'VRESM-1-0']
```
Would you consider the registration of `INM-CM5-0` to bring this closer to the similar format of existing entries?

**Comment by volodin1** [comment]
Dear Paul,
Model name was changed from INMCM50 to INM-CM5-0,  and must I also change
other attributes (label, label_extended, ...) or change of model name is
enough? Similar question about INM-CM4-8

**Comment by durack1** [comment]
@volodin1 up to you, the key `INM-CM5-0` is the primary handle that will make up the directory and file name information, with the sub-components less important for files and directory naming. For consistency, you can amend whatever you like.

I also note that I have not taken your registration information above and merged that across into the source_id.json file - once you have finalised the details I will do that (merge your changes) and then close this issue (which I have just reopened)

**Comment by davidhassell** [comment]
Hello,

My name is David Hassell and I am part of the ES-DOC team that is working on CMIP6 documentation. As a part of this process, we need a liaison at INM who can act as the interface between the scientists who actually provide the documentation and the ES-DOC team. I was wondering if you are able to help with identifying such a person?

The documentation process has changed considerably since CMIP5 (in response to the feedback that we received), and you can now choose from a variety of documentation methods. Our aim is to train the ES-DOC liaison who can then work with people at INM. No specialist knowledge about ES-DOC nor climate modelling is required, but neither would it be a problem, of course. Some institutes have appointed a member of the modelling team, others not.

I would be very grateful f you could let me know by e-mail (david.hassell at ncas.ac.uk) if you have any suggestions.

All the best,

David Hassell

National Centre for Atmospheric Science
University of Reading
UK

**Comment by durack1** [comment]
@volodin1 your model is registered, take a look at [CMIP6_source_id.html](http://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_source_id.html) and let me know if any further changes are required

**Comment by volodin1** [comment]
Dear David,

Andrey Gritsoun (asgrit@mail.ru) could be liaison person from INM.

Best regadrs, Evgeny

**Comment by durack1** [comment]
@volodin1 we noted a formatting inconsistency with the `aerosol`, `atmos`, `land`, `ocean` and `seaIce` `nominal_resolution` values for `INM-CM5-0`, and have now corrected this. Please check your complete model registration details in [CMIP6_source_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_source_id.html) and get back to me with any further tweaks required.

For reference, this issue was caught and resolved in WCRP-CMIP/CMIP6_CVs#466
